Modernize (#11)
digitalbazaar/bedrock-redis: CHANGELOG.md

# bedrock-redis ChangeLog

## 4.0.0 - 2022-xx-xx

> ### Changed

Reviewer: ```suggestion
### Changed
- **BREAKING**: Rename package from `bedrock-redis` to `@bedrock/redis`.
```

> - **BREAKING** Convert to esm.

Reviewer: We (I) had been using common text for this. Not a big deal, but could be written as:
```suggestion
- **BREAKING** Convert to module (ESM).
```

- **BREAKING** Update redis client to v4, use node-redis directly.
- **BREAKING** Use namespaced and up to date bedrock packages throughout.
- **BREAKING** Remove prefix option, update backoff options.

> - **BREAKING** Change min/max config options to minTimeout/maxTimeout.

Reviewer: Be more explicit with what needs updating:
```suggestion
- **BREAKING** Remove `prefix` option.
- **BREAKING** Update retry options due to changing from `backo` package to `retry`:
  - Rename `bedrock.config.redis.retry` to `bedrock.config.redis.retryOptions`.
  - Rename `min`/`max` to `minTimeout`/`maxTimeout`.
  - Rename `maxTimesConnected` to `retries`.
  - Remove `jitter` and `maxTotal`.
```

> 

Reviewer: Some more changes related to other comments and updates:
```suggestion
- **BREAKING**: Require Node.js >=16.
- Use `package.json` `files` field.

### Added
- Lint code.
- Add tests.

```
